openbas-api/src/main/java/io/openbas/rest/inject_test_status/InjectTestStatusApi.java

```diff
@@ -0,0 +1,42 @@
+package io.openbas.rest.inject_test_status;
+
+import io.openbas.database.model.InjectTestStatus;
+import io.openbas.rest.helper.RestBehavior;
+import io.openbas.service.InjectTestStatusService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/inject_test_status")
```

Comment:
I'm not a fan of this url. I think it's great that you switch for a dedicated controller but you should keep using the INJECT_URI from InjectApi.java